data/sql/base/db_world/command.sql

```diff
@@ -39,6 +39,7 @@ INSERT INTO `command` (`name`, `security`, `help`) VALUES
 	('account password', 0, 'Syntax: .account password $old_password $new_password $new_password [$email]\r\n\r\nChange your account password. You may need to check the actual security mode to see if email input is necessary.'),
 	('account set', 2, 'Syntax: .account set $subcommand\nType .account set to see the list of possible subcommands or .help account set $subcommand to see info on subcommands'),
 	('account set 2fa', 0, 'Syntax: .account set 2fa <account> <secret/off>\nProvide a base32 encoded secret to setup two-factor authentication for the account.\nSpecify \'off\' to disable two-factor authentication for the account.'),
+	('account set email', 4, 'Syntax: .account set email <account> <email> <email confirmation>\nAdds an email to the account.'),
```

Comment:
Don't edit this file, it gets updated automatically once we update the database version on a new release

Reply:
ah, do i need to update SQL anywhere else for this command?

Reply:
you need to create a new file inside /updates/db_world/ with the insert query for it

Reply:
@Nyeriah I think i got that updated, please confirm

Reply:
Looks good. To note is that the name of the file within /db_world/ is trivial as it will be renamed by CI when pushed